modules/books.js

```diff
@@ -0,0 +1,67 @@
+const saveBooks = (library) => {
+  localStorage.setItem('myAwesomeBooks', JSON.stringify(library.awesomeBooks));
+};
+
+const displayBooks = (library) => {
+  const booksList = document.querySelector('.books');
+  booksList.innerHTML = '';
+  let i = 0;
+  library.awesomeBooks.forEach((book) => {
+    // eslint-disable-next-line no-plusplus
+    i++;
+    const bookElement = document.createElement('div');
+    bookElement.classList.add('book');
+    if (i % 2 !== 0) {
+      bookElement.classList.add('odd-color');
+    }
+    const headTitle = document.createElement('h2');
+    headTitle.textContent = `"${book.title}" by ${book.author}`;
+
+    const removeBtn = document.createElement('button');
+    removeBtn.textContent = 'Remove';
+
+    removeBtn.addEventListener('click', () => {
+      library.removeBook(book.id);
+      displayBooks();
+    });
```

Comment:
- [ ] Kindly here you have to give the `displayBooks();` method, the `library` parameter, because it takes it, and it makes the remove function not work.
Like  `displayBooks(library);`